cmd/ndm_daemonset/filter/osdiskexcludefilter_test.go

```diff
@@ -118,6 +118,11 @@ func TestOsDiskExcludeFilterExclude(t *testing.T) {
 			disk:     &controller.DiskInfo{Path: "/dev/vg0-lv0"},
 			expected: false,
 		},
+		"exclude path is empty and device path is /dev/sda": {
+			filter:   oSDiskExcludeFilter{excludeDevPath: ""},
+			disk:     &controller.DiskInfo{Path: "/dev/sda"},
+			expected: true,
```

Comment:
What does `true` mean? The disk will be excluded or not excluded?

Reply:
true means filter passed, and device will not be excluded.

Reply:
should we rename `expected` to -- `diskExcluded` ?

Reply:
the comments on the function have been updated.